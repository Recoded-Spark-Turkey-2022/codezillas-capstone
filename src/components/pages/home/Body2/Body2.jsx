import React from 'react'
import { useNavigate } from 'react-router-dom'
import People from './image/people.png'

const Body2 = () => {

  const navigate = useNavigate()

  function handleClick(){
    navigate('/book')
  }

  return (
    <div className='w-full h-screen bg-amber-100 flex justify-center items-center'>
      <div className='w-8/12 h-5/6 border flex flex-col items-start justify-evenly '>
        <div className="h-3/6 flex flex-col justify-evenly items-start gap-10">
          <h4 className='text-5xl font-medium uppercase '>Professional, licensed, and vetted therapists that you can trust</h4>

          <div>
              
              <img src={People} alt="People" />
          </div>

          <div className="flex">
            <p>Tap into the world`s largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.</p>
          </div>

        </div>

        <div className="h-2/6 flex justify-center items-start">
        <input 
              type="button" 
              value="BOOK AN APPOINTMENT" 
              className=' py-2 px-5 hover:bg-cyan-200 bg-cyan-400 font-medium rounded text-sm md:text-lg'
              onClick={() => handleClick()}/>
        </div>
      </div>
      
    </div>
  )
}

export default Body2