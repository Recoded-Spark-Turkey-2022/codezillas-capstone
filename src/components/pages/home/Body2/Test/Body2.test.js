import { Provider } from "react-redux"
import { BrowserRouter } from "react-router-dom"
import renderer from "react-test-renderer"
import Body2 from "../Body2"
import {store} from "../../../../../app/store"



it(`renders correctly`, () => {
    const Body2comp = renderer.create(
        <Provider store={store} >
            <BrowserRouter>
                <Body2 />
            </BrowserRouter>
        </Provider>
        
    )
    .toJSON()
        expect(Body2comp).toMatchSnapshot()
    
})