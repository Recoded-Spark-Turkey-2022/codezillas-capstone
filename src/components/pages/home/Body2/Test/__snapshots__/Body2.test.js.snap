// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="  bg-amber-200 flex flex-col justify-center items-center max-w-[ 1440px] min-w-[500px]"
>
  <div
    className="w-8/12 h-screen  flex flex-col items-start justify-evenly  "
  >
    <div
      className="h-4/6 flex flex-col justify-evenly items-start gap-10 min-w-fit "
    >
      <h4
        className=" w-3/4  text-5xl font-medium uppercase font-sans  "
      >
        Professional, licensed, and vetted therapists that you can trust
      </h4>
      <div
        className="flex"
      >
        <img
          alt="People"
          src="people.png"
        />
      </div>
      <div
        className="flex"
      >
        <p
          className=" font-sans font-medium   md:text-lg"
        >
          Tap into the world\`s largest network of licensed, accredited, and experienced therapists who can help you with a range of issues including depression, anxiety, relationships, trauma, grief, and more. with our therapists, you get the same professionalism and quality you would expect from an in-office therapist, but with the ability to communicate whenever and however you want.
        </p>
      </div>
    </div>
    <div
      className="flex flex-col items-start justify-evenly gap-5 sm:-mt-10 "
    >
      <input
        className="  py-3 px-5 hover:bg-cyan-200 bg-cyan-400 font-medium rounded text-lg md:text-lg"
        onClick={[Function]}
        type="button"
        value="BOOK AN APPOINTMENT"
      />
    </div>
  </div>
</div>
`;
